src/pagination/get_data.jsx

```diff
@@ -0,0 +1,544 @@
+        cost: '2.500.000',
+        oldCost: '',
+        new: false,
+        discount: ''
+    },
+    {
+        path: 'lolito.png',
+        name: 'Lolito27',
+        type: 'Luxury big sofa',
+        cost: '7.000.000',
+        oldCost: '14.000.000',
+        discount: '-50%'
+    },
+    {
+        path: 'muggo.png',
+        name: 'Muggo28',
+        type: 'Small mug',
+        cost: '150.000',
+        oldCost: '',
+        new: true,
+        discount: ''
+
+    },
+    {
+        path: 'pingky.png',
+        name: 'Pingky29',
+        type: 'Cute bed set',
+        cost: '7.000.000',
+        oldCost: '14.000.000',
+        discount: '-50%',
+        new: false,
+    },
+    {
+        path: 'potty.png',
+        name: 'Potty30',
+        type: 'Minimalist flower pot',
+        cost: '500.000',
+        oldCost: '',
+        new: true,
+        discount: ''
+    },
+    {
+        path: 'respira.png',
+        name: 'Respira31',
+        type: 'Outdoor bar table and stool',
+        cost: '500.000',
+        oldCost: '',
+        new: true,
+        discount: ''
+    },
+    {
+        path: 'syltherine.png',
+        name: 'Syltherine32',
+        type: 'Stylish cafe chair',
+        cost: '2.500.000',
+        oldCost: '3.500.000',
+        discount: '-30%',
+        new: false
+    },
+    {
+        path: 'grifo.png',
+        name: 'Grifo33',
+        type: 'Night lamp',
+        cost: '2.500.000',
+        oldCost: '',
+        new: false,
+        discount: ''
+    },
+    {
+        path: 'leviosa.png',
+        name: 'Leviosa34',
+        type: 'Stylish cafe chair',
+        cost: '2.500.000',
+        oldCost: '',
+        new: false,
+        discount: ''
+    },
+    {
+        path: 'lolito.png',
+        name: 'Lolito35',
+        type: 'Luxury big sofa',
+        cost: '7.000.000',
+        oldCost: '14.000.000',
+        discount: '-50%'
+    },
+    {
+        path: 'muggo.png',
+        name: 'Muggo36',
+        type: 'Small mug',
+        cost: '150.000',
+        oldCost: '',
+        new: true,
+        discount: ''
+
+    },
+    {
+        path: 'pingky.png',
+        name: 'Pingky37',
+        type: 'Cute bed set',
+        cost: '7.000.000',
+        oldCost: '14.000.000',
+        discount: '-50%',
+        new: false,
+    },
+    {
+        path: 'potty.png',
+        name: 'Potty38',
+        type: 'Minimalist flower pot',
+        cost: '500.000',
+        oldCost: '',
+        new: true,
+        discount: ''
+    },
+    {
+        path: 'respira.png',
+        name: 'Respira39',
+        type: 'Outdoor bar table and stool',
+        cost: '500.000',
+        oldCost: '',
+        new: true,
+        discount: ''
+    },
+    {
+        path: 'syltherine.png',
+        name: 'Syltherine40',
+        type: 'Stylish cafe chair',
+        cost: '2.500.000',
+        oldCost: '3.500.000',
+        discount: '-30%',
+        new: false
+    },
+]
+
+const productHooverItems = [
```

Comment:
```suggestion
const productHoverItems = [
```
`productActions` is even better